Chapter1/Zad5.py

```diff
@@ -0,0 +1,38 @@
+'''
+Created on 28 sty 2018
+
+@author: agata.wiewiora
+'''
+
+
+def count_tables_amount (students_amount):
+    table_students = 2 
+    temp = students_amount % table_students
```

Comment:
unikaj nazw takich jak "temp", bo to nic nie mówi, całe działanie modulo można wrzucić do ifa